ui/shared/components/panel/variants/VariantIndividuals.jsx

```diff
@@ -112,12 +112,16 @@ export const Alleles = React.memo(({ genotype, variant, isHemiX, warning }) =>
         content={<div><b>Warning:</b> {warning}</div>}
       />
     }
-    {genotype.numAlt >= 0 ?
+    {!variant.svType && genotype.numAlt >= 0 ?
       <Header.Content>
         <Allele isAlt={genotype.numAlt > (isHemiX ? 0 : 1)} variant={variant} textAlign="right" />
         /{isHemiX ? '-' : <Allele isAlt={genotype.numAlt > 0} variant={variant} textAlign="left" />}
       </Header.Content> :
-      <Header.Content>CN: {genotype.cn === (isHemiX ? 1 : 2) ? genotype.cn : <b><i>{genotype.cn}</i></b>}</Header.Content>
+      <Header.Content>
+        {genotype.cn && <span>CN: {genotype.cn === (isHemiX ? 1 : 2) ? genotype.cn : <b><i>{genotype.cn}</i></b>}</span>}
+        {variant.svType &&
+        <span>,{genotype.numAlt > 0 ? ' X' : ' -'}/{isHemiX || genotype.numAlt < 2 ? '-' : 'X'}</span>}
```

Comment:
for exome SVs there isn't a `numAlt` at all so we need to not show this section in those cases

Reply:
Got it.